src/App.js

```diff
@@ -1,16 +1,44 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import Clock from "./components/Clock";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
+          <h1>WORLD CLOCK</h1>
+          <Container fluid>
+            <Row>
```

Comment:
the rows are being repeated here, it is possible to create a component to display each row instead.

that component would probably take a prop for `timeZone`, since that's the only value that differs on each row

Reply:
![image](https://user-images.githubusercontent.com/30802551/191047615-45cad9f5-2124-4b6a-aec7-273f1100734b.png)
is this what you mean here?

Reply:
![image](https://user-images.githubusercontent.com/30802551/191047748-b77943fb-ee15-4eb5-abe0-f68ef116aa1d.png)

Reply:
yeah, that's much better, don't you think?

one more step you can take is to use a `.map` to map the component out.

i would recommend giving a name to your component that better reflects the context in which it is used. e.g. you can call it `TimezoneRow`.